src/settings/AudioSettings.tsx

```diff
@@ -125,20 +134,41 @@ export default function GeneralSettings(props: ISettingsPanelProps) {
       color: '#bb4220',
     },
   };
+  const switchStyle = {
+    '& .MuiSwitch-switchBase': {
+      '&.Mui-checked': {
+        color: '#fff',
+        '+.MuiSwitch-track': {
+          backgroundColor: '#bb4220',
+          opacity: 1.0,
+        },
+      },
+      '&.Mui-disabled + .MuiSwitch-track': {
+        opacity: 0.5,
+      },
+    },
+  };
+  const headingStyle = {
+    color: '#fff',
+    my: '1em',
+  };
 
   return (
     <Stack
       component="form"
       sx={{
-        '& > :not(style)': { m: 0, width: '50ch' },
+        '& > :not(style)': { width: '50ch' },
       }}
       noValidate
       autoComplete="off"
     >
+      <Box component="h3" sx={headingStyle}>
```

Comment:
Minor gripe - I don't really like the look of this text (does it need to be here?)

Reply:
Not really, I added the headers to differentiate the setting types but it's probably overkill for the amount of audio settings we currently have. I'll remove them and their styling variable.